cfme/tests/v2v/test_v2v.py

```diff
@@ -1,9 +1,97 @@
+    pytest.mark.meta(server_roles=['+git_owner'])
+]
+
+
+def pytest_generate_tests(metafunc):
+    argnames1, argvalues1, idlist1 = testgen.providers_by_class(metafunc, [VMwareProvider])
+    argnames2, argvalues2, idlist2 = testgen.providers_by_class(metafunc, [RHEVMProvider])
+
+    new_idlist = []
+    new_argvalues = []
+    new_argnames = ['nvc_prov', 'rhvm_prov']
+
+    for index1, argvalue_tuple1 in enumerate(argvalues1):
+        for index2, argvalue_tuple2 in enumerate(argvalues2):
+            new_idlist.append('{}-{}'.format(idlist1[index1], idlist2[index2]))
+            new_argvalues.append((argvalue_tuple1[0], argvalue_tuple2[0]))
+    testgen.parametrize(metafunc, new_argnames, new_argvalues, ids=new_idlist, scope="module")
+
+
+@pytest.fixture(scope='module')
+def initial_setup(appliance, request, nvc_prov, rhvm_prov):
+    """
+    Steps:
+        1.Validate host credentials
+        2.Assign tags to conversation host
+        3.Import automate domain via github repo
+    """
+
+    # Validating host credentials
+    test_host = random.choice(rhvm_prov.data["hosts"])
+    for host_obj in conf.cfme_data.get('management_systems', {})[rhvm_prov.key].get('hosts', []):
+        if test_host.name == host_obj['name']:
+            host_data = host_obj
+    host_collection = appliance.collections.hosts
+    host_obj = host_collection.instantiate(name=test_host.name, provider=rhvm_prov)
+    with update(host_obj, validate_credentials=True):
+        host_obj.credentials = host.get_credentials_from_config(host_data['credentials'])
+
+    # Assigning tags to conversation host
+    tag1 = Tag(display_name='t', category=Category(display_name='V2V - Transformation Host *'))
+    tag2 = Tag(display_name='VDDK', category=Category(display_name='V2V - Transformation Method'))
+    tag1_bool = (
+        (tag1.display_name, tag1.category.display_name) in
+        [(tags.display_name, tags.category.display_name) for tags in host_obj.get_tags()[0]]
+    )
+    tag2_bool = (
+        (tag2.display_name, tag2.category.display_name) in
+        [(tags.display_name, tags.category.display_name) for tags in host_obj.get_tags()[1]]
+    )
+    if not (tag1_bool or tag2_bool):
+        host_obj.add_tags(tags=(tag1, tag2))
+
+    # Importing automate domain via github repo
+    repo = AutomateGitRepository(url="https://github.com/fdupont-redhat/v2v-automate.git",
+                                 verify_ssl=True,
+                                 appliance=appliance)
+    domain = repo.import_domain_from(**{'branch': 'origin/extended_reporting_and_rhv_upload'})
+    request.addfinalizer(domain.delete_if_exists)
+
+
+vms = []
+
+
+@pytest.fixture(scope="module")
+def vm_list(appliance, nvc_prov):
+    """Fixture to provide list of vm objects"""
+    # TODO: add template names and configuration in yaml and remove following code
+    templates = [nvc_prov.data.templates.small_template['name']]
+    for template in templates:
```

Comment:
**REQUIRED**
Why to not use https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/vm.py#L59? Just add `small_template_vm`

Reply:
@Yadnyawalkya @quarckster small_template should not be used, I believe it is either PhotonOS or damnSmallLinux in our data.yml, which is not supported by v2v.